icommons_ext_tools/requirements/demo.txt

```diff
@@ -9,6 +9,6 @@ newrelic==2.8.0.7
 gunicorn
 
 # Note: In the demo environment we use the /master branch
-git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@master#egg=django-icommons-common
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v1.9.5#egg=django-icommons-common
```

Comment:
I get the impression this is part of a larger discussion, but what makes icommons-common need to be tagged, not pointed to master, when the other dependencies don't?